### PR TITLE
[MD] Friday tiles

### PR DESCRIPTION
**Pull request by AlbertTheTerrible**
#### Summary
MD "added status icons and more"

#### Content of the change
Added status icons, still testing their size, for now they'll remain 16x16 at their max (they're mostly half of that)
Added stairs, that can stack vertically 👍 
(they look pretty bad on dirt and everywhere else, i'll add more kinds of stairs later)
Added metal lockers and several variations
Fixed the oftet on the vertical door and added their proper variations

These sprites still need their proper id's i'll come back to it later. If anyone wants to get on that, be my guest 😂

### CODE REVIEW

**Comment by AlbertTheTerrible** [comment]
Added so many jsons and no bugs with jsons 🎃 

I'm betting none of them works

**Comment by AlbertTheTerrible** [comment]
Yep i knew it

**Comment by AlbertTheTerrible** [comment]
I think this is a big enough pr, i'll be looking to merge if everything is ok and pull a new one with more stuff ;)

File: gfx/Mushroom-Dream/pngs_tiny_16x16/weather/rain.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"id": "weather_rain_drop",
+	"bg": ["rain", "rain2"]
```
Comment (Fris0uman) [inline gfx/Mushroom-Dream/pngs_tiny_16x16/weather/rain.json:3]:
```suggestion
  "bg": [
    { "weight": 1, "sprite": "rain" },
    { "weight": 1, "sprite": "rain2" }
  ]
```
Are you sure you don't need something like this in the json?

File: gfx/Mushroom-Dream/pngs_tall_32x48/furniture/f_locker/f_locker.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+"id": "f_locker",
+"fg": [
+	{ "weight": 1, "sprite": "f_locker1" },
+    { "weight": 1, "sprite": "f_locker2" },
+    { "weight": 1, "sprite": "f_locker3" },
+    { "weight": 1, "sprite": "f_locker4" }
+	],
+	"rotates": false
+}
```
Comment (Fris0uman) [inline gfx/Mushroom-Dream/pngs_tall_32x48/furniture/f_locker/f_locker.json:10]:
```suggestion
{
  "id": "f_locker",
  "fg": [
    { "weight": 1, "sprite": "f_locker1" },
    { "weight": 1, "sprite": "f_locker2" },
    { "weight": 1, "sprite": "f_locker3" },
    { "weight": 1, "sprite": "f_locker4" }
  ],
  "rotates": false
}

```
Linting json

**Review by Fris0uman — COMMENTED** [review]
- woodTile1 to 7 need some json in order to be actually used
- 5.png needs either a proper id or to move to the temp folder
The rest looks good to me

**Comment by AlbertTheTerrible** [comment]
i moved those tiles with improper ids to temp, i'll re name them later

**Pull request merged**